shell.nix

```diff
@@ -0,0 +1,18 @@
+{ versions ? import ./nix/versions.nix
```

Comment:
Can we please whether rust analyzer and clippy is installed in this shell?

Reply:
It is installed by default.

Reply:
Moreover, `nix-shell` should provide you most common tools for developing our project -  cargo, rust, rust-analyzer, clippy, rust-src, clang, clang++.